views/edit.erb

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>edit - memo app.</title>
+    <link rel="stylesheet" href="/style.css">
+  </head>
```

Comment:
sinatraにはlayout機能があります。このあたりの定型的なHTMLはまとめて1ファイルにしてしまいましょう。
あまり公式にはドキュメントがちゃんと書かれてないですが、
https://sinatrarb.com/intro.html#views--templates

に

> This will render views/index.erb embedded in the views/post.erb (default is views/layout.erb, if it exists).

という記載がある通り、 `views/layout.erb` を作ってあげるとデフォルトでそれを読みこんでくれます。